umaserverdemo/src/main/kotlin/com/lightspark/Vasp2.kt

```diff
@@ -163,18 +163,15 @@ class Vasp2(
         }
 
         val receivingCurrency = getReceivingCurrencies(senderUmaVersion)
-            .firstOrNull { it.code == payreq.receivingCurrencyCode() } ?: run {
-            call.respond(HttpStatusCode.BadRequest, "Unsupported currency.")
-            return "Unsupported currency."
-        }
+            .firstOrNull { it.code == payreq.receivingCurrencyCode() }
```

Comment:
I think we should only allow this if the receiving currency code is null. If the sender is sending an invalid currency, that should still be an error.

Reply:
okk ya that makes sense!